doc/examples/perception_pipeline/perception_pipeline_tutorial.rst

```diff
@@ -1,198 +1,96 @@
-:moveit1:
-
-..
-   Once updated for MoveIt 2, remove all lines above title (including this comment and :moveit1: tag)
-
 Perception Pipeline Tutorial
-============================
+==================================
+
+Introduction
+------------
+MoveIt serves sensor plugins for utilizing 3D sensor data in planning pipeline. When it's ran the demo of this tutorial, it should be seen the following rviz screen.
 
-MoveIt allows for seamless integration of 3D sensors using `Octomap <http://octomap.github.io/>`_.
-Once properly configured, you should see something like this in rviz:
+.. raw:: html
 
-.. image:: perception_configuration_demo.png
-   :width: 700px
+        <iframe width="560" height="315" src="https://www.youtube.com/embed/bpMHFxcbRXg" title="MoveIt Perception Pipeline Demo" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
 
 Getting Started
 ---------------
-If you haven't already done so, make sure you've completed the steps in :doc:`Getting Started </doc/tutorials/getting_started/getting_started>`.
-
-Configuration
```

Comment:
Why are you deleting all this, is it outdated?

Reply:
I actually didn't delete it. I just made some modifications. You can find the relavant redirection in this link .

https://github.com/CihatAltiparmak/moveit2_tutorials/blob/e7eec45b14d360b8e8ffe2471b38f22e9f7e05cf/doc/examples/perception_pipeline/perception_pipeline_tutorial.rst?plain=1#L15-L18